pandas/_libs/missing.pyx

```diff
@@ -220,6 +220,51 @@ cpdef ndarray[uint8_t] isnaobj(ndarray arr):
     return result.view(np.bool_)
 
 
+@cython.wraparound(False)
+@cython.boundscheck(False)
+cpdef ndarray[uint8_t] isna_string(ndarray arr):
```

Comment:
Can we return a const uint8_t memory view instead of an ndarray? I still think generally better to use memoryviews over the long term to stay somewhat backend agnostic

Reply:
I think for the return value we want a numpy array? (inside the function we could use more memoryviews)  
I am actually only using this function from python, so can make this a `def` function and remove the return type